src/Automation/ScanTools.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+
+namespace Axe.Windows.Automation
+{
+    class ScanTools : IScanTools
+    {
+        public IOutputFileHelper OutputFileHelper { get; }
+        public IScanResultsAssembler ResultsAssembler { get; }
+        public ITargetElementLocator TargetElementLocator { get; }
+
+        public ScanTools(IOutputFileHelper outputFileHelper, IScanResultsAssembler resultsAssembler, ITargetElementLocator targetElementLocator)
+        {
+            if (outputFileHelper == null) throw new ArgumentNullException(nameof(outputFileHelper));
+            if (resultsAssembler == null) throw new ArgumentNullException(nameof(resultsAssembler));
+            if (targetElementLocator == null) throw new ArgumentNullException(nameof(targetElementLocator));
+            this.OutputFileHelper = outputFileHelper;
```

Comment:
>this. [](start = 12, length = 5)

Can we remove the unnecessary "this." prefixes? This is a really old coding pattern going back into the Visual Studio code base (and into the Exchange code base before that). This pattern is discouraged by Roslyn checks and I'd love to see us stop using it.

Reply:
You'll notice I don't do that very often. I did it in this case because the property names are exactly the same as the type names. But I have removed them because it sounds like they're no longer welcome in the modern world.